src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -168,11 +182,11 @@ private void OnOutputGroupChanged(IImmutableDictionary<string, IOutputGroup> e)
             }
         }
 
-        private void OnChanged(IProjectVersionedValue<Tuple<IProjectSubscriptionUpdate, IProjectImportTreeSnapshot, IProjectSubscriptionUpdate, IImmutableDictionary<string, IOutputGroup>>> e)
+        private void OnChanged(IProjectVersionedValue<Tuple<IProjectSubscriptionUpdate, IProjectImportTreeSnapshot, IProjectSubscriptionUpdate, IImmutableDictionary<string, IOutputGroup>, IProjectItemSchema>> e)
```

Comment:
Tuples should work, but the callers need to be updated to pass tuples/System.ValueTuple instead of System.Tuple. That may have been the problem.
I tried to stop by your office to look at the error. Ping me tomorrow when you have time to try again.

---
In reply to: [121548234](https://github.com/dotnet/project-system/pull/2416#discussion_r121548234) [](ancestors = 121548234)